.gitignore

```diff
@@ -1,3 +1,4 @@
 webserv
 *.o
-objs/
\ No newline at end of file
+objs/
+settings.json
```

Comment:
why not *.json in genreal ? will we use some json files?

Reply:
Why and where would we use jdon files?